### PR TITLE
Allocate random (sequential)

### PR DESCRIPTION
**Pull request by dror-stratoscale**
Quick and simple solution to asset problem: allocate starting from a random point and not from beginning

### CODE REVIEW

**Comment by dror-stratoscale** [comment]
@mickey-stratoscale

**Comment by mickey-stratoscale** [comment]
@rom-stratoscale @eliran-stratoscale please review

File: asset/server/allocations.py [hunk 1]
```diff
@@ -28,11 +29,12 @@ def create(self, assetKind, assetCount, pool, continent, allocationInfo):
             raise Exception(
                 "Continent %(continent)s assetKind %(assetKind)s pool %(pool)s does not have enough "
                 "available resources" % dict(continent=continent, assetKind=assetKind, pool=pool))
+        allocStart = random.randint(0, len(poolInstance['assets'])-assetCount)
```
Comment (igal-stratoscale) [inline asset/server/allocations.py]: didn't you missed len(list)-1 here?
len(poolInstance['assets'])-assetCount -1

Comment (dror-stratoscale) [inline asset/server/allocations.py]: Slice operator ([::]) syntax makes it work
For example if assetCount == len(poolinstance) (a valid allocation) then we call randint(0,0) which returns 0 and the we do assets=poolInstance[0:len(poolinstance)] which is OK.
Also if assetCount == 1, we call randint(0, len(poolinstance)-1), worst case this returns len(poolinstance)-1 so we have assets=poolinstance[len(poolinstance)-1:len(poolinstance)], also OK

**Pull request merged**